packages/terra-collapsible-menu-view/src/CollapsibleMenuViewItemGroup.jsx

```diff
@@ -26,10 +26,6 @@ const propTypes = {
    * A list of keys of the CollapsibleMenuView.Items that should be selected.
    */
   selectedKeys: PropTypes.arrayOf(PropTypes.string),
-  /**
-   * String that labels the group for accessibility.
-   */
-  ariaLabel: PropTypes.string,
```

Comment:
I think we still need this prop along with this prop we need to create a prop for GroupId so that it can be used in menu item for `aria-describedby` attribute.

Reply:
Updated - https://github.com/cerner/terra-framework/pull/1841/commits/4cf44699ba0fff5480d30bd3f25b1b708a7b6fa6